simpleflow/job/k8s.py

```diff
@@ -0,0 +1,73 @@
+from base64 import b64encode
+from jinja2 import Template
+import json
+import os
+import yaml
+
+import kubernetes.config
+import kubernetes.client
+
+from simpleflow.utils import json_dumps
+
+
+class KubernetesJob(object):
+    def __init__(self, job_name, domain, response):
+        self.job_name = job_name
+        self.response = response
+        self.domain = domain
+
+    def load_config(self):
+        """
+        Load config in the current Kubernetes cluster, either via in cluster config
+        or via the local kube config if on a development machine.
+        """
+        try:
+            kubernetes.config.load_incluster_config()
+        except:
```

Comment:
PEP8: do not use bare `except`

Reply:
_Touché_, for once PEP8 will improve my code :p